src/dackar/text_processing/Abbreviation.py

```diff
@@ -61,9 +61,13 @@ def abbreviationSub(self, text):
       corrected = sent
       splitSent = sent.split()
       for word in splitSent:
-        if word not in not_acronyms:
-          if word in self.abbrDict.keys():
-            full = self.abbrDict[word]
+        splitWord = re.split(r'[-\d+]', word) # split word if word contains '-' or numbers
```

Comment:
should we perform this treatment before the abbreviation handling (in the text pre-processing steps)?

Reply:
your are right, fixed.